src/main/java/seedu/address/model/company/Phone.java

```diff
@@ -66,7 +55,7 @@ public boolean equals(Object other) {
         }
 
         Phone otherPhone = (Phone) other;
-        return standardizeNumber(value).equals(standardizeNumber(otherPhone.value));
+        return value.equals(otherPhone.value);
```

Comment:
This means that 1234 5678 is not equal to 12345678? 
If I understood wrongly then just ignore